Rework Quickstart guide (#1193)
scality/metalk8s: docs/quickstart/setup.rst

Setup of the environment
========================

.. _MetalK8s: https://github.com/scality/metalk8s
.. _CentOS: https://www.centos.org
.. _RHEL: https://access.redhat.com/products/red-hat-enterprise-linux

General requirements
--------------------

> MetalK8s_ clusters run on machines (including VMs) with CentOS_ / RHEL_ 7.6 or

Reviewer: > MetalK8s_ clusters require machines running CentOS_ / RHEL_ 7.6 or higher as their operating system. These machines may be virtual machines or physical machines, with no difference in setup procedure.

higher for their operating system.

> For this quickstart, we will need 5 machines (or 3, if you want to run workload

Reviewer: ```suggestion
For this quickstart, we will need 5 machines (or 3, if running workload
```

applications on your control-plane nodes).

Sizing
^^^^^^

> Each machine should have at least 2 CPU cores, 4 GB of RAM, and a root

Reviewer 2: s/root filesystem/root partition/ ?

filesystem larger than 40 GB.

For sizing recommendations depending on sample use cases, see the
:doc:`Installation guide </installation-guide/sizing>`.

Proxies
^^^^^^^
For nodes operating behind a proxy, add the following lines to each cluster

> server's :file:`/etc/environment` file:

Reviewer: ```suggestion
member's :file:`/etc/environment` file:
```
